test/passing/let_in_constr.ml

```diff
@@ -1 +1,4 @@
-let _ = Some (let open! List in 1)
+let _ =
+  Some
+    (let open! List in
+    1)
```

Comment:
Should `1` be aligned with the `l` in `let` here?

Reply:
Quite possibly, but that is unrelated so won't delay this PR.